src/main/java/com/assure/movie/model/domain/Movie.java

```diff
@@ -29,10 +29,10 @@
     @Enumerated(EnumType.STRING)
     private Rate rate;
```

Comment:
Since you have this column as an Enum :), what will happen if someone inserts manually a record with some value that does not belong to this Enum and you want to get the data from this table, let's say using the findAll jpa method?

Reply:
I did a fast test and an exception is throw because it's not posible to map that value with a enum field.

Reply:
Perfect, thanks!